Core/Extensions/ImageLinkTagHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace WebNetSample.Core.Extensions;
+
+[HtmlTargetElement("webnetsample-imagelink", TagStructure = TagStructure.NormalOrSelfClosing)]
+public class ImageLinkTagHelper : TagHelper
+{
+    [HtmlAttributeName("image-id")]
+    public string? Id { get; set; }
+
+    public override void Process(TagHelperContext context, TagHelperOutput output)
+    {
+        output.TagName = "a"; // Replaces <websamplenet-imagelink> with <a> tag
```

Comment:
Guess we don't need this comment since this code row describes itself clearly :)

Reply:
That was just for me to understand the underlying logic but I'll remove it.